gips/data/prism/prism.py

```diff
@@ -282,59 +287,50 @@ def get_bil_file(d, a):
                     # default lag of 3 days SB configurable.
                     lag = 3
                     fname = re.sub(r'\.tif$', '-{}.tif'.format(lag), fname)
-                    VerboseOut('Using default lag of {} days.'.format(lag), 2)
+                    utils.verbose_out('Using default lag of {} days.'.format(lag), 2)
 
                 if os.path.exists(fname):
                     os.remove(fname)
 
-                inv = self.inventory(
-                    dates='{},{}'.format(
-                        datetime.strftime(
-                            self.date - timedelta(days=lag - 1), '%Y-%m-%d',
-                        ),
-                        datetime.strftime(self.date, '%Y-%m-%d'),
+                date_spec = '{},{}'.format(
+                    datetime.strftime(
+                        self.date - timedelta(days=lag - 1), '%Y-%m-%d',
                     ),
+                    datetime.strftime(self.date, '%Y-%m-%d'),
+                )
+                inv = self.inventory(
+                    dates=date_spec,
                     products=['ppt'],
                 )
                 inv.process()
-                inv = self.inventory(
-                    dates='{},{}'.format(
-                        datetime.strftime(
-                            self.date - timedelta(days=lag - 1), '%Y-%m-%d',
-                        ),
-                        datetime.strftime(self.date, '%Y-%m-%d'),
-                    ),
+                inv = self.inventory(  # because DataInventory object doesn't update
+                    dates=date_spec,
                     products=['ppt'],
                 )
                 if len(inv.data) < lag:
-                    raise Exception(
-                        key + ': requires {} preceding days ppt assets.'
-                        .format(lag) +
-                        '\nOnly {} days found.'.format(len(inv.data))
+                    utils.verbose_out(
+                        '{}: requires {} preceding days ppt ({} found).'
+                        .format(key, lag, len(inv.data)),
+                        3,
                     )
+                    continue
```

Comment:
optional:  this seems to be 60ish lines away from its `for` statement, so it might be worth it to put a comment here saying which loop context is relevant